### ISSUE TITLE
null is not an object (evaluating 'NativeDatabaseBridge[MethodName]')

### ISSUE THREAD

**Issue by piolAtif** [Nozbe/WatermelonDB#677]
Hi guys,
Getting error of `null is not an object (evaluating 'NativeDatabaseBridge[MethodName]') watermelnodb` on ios simulator 11.1
but for android simulator watermelondb installation is working.
<img width="369" alt="Screenshot 2020-05-13 at 4 12 52 PM" src="https://user-images.githubusercontent.com/15261138/81803184-f33e3600-9534-11ea-8791-6eafc4beb535.png">

Checked on other issues: [mostly related to header file in ios ] 
That is already added.

Tried to use command `pod install` 
getting error
<img width="1440" alt="Screenshot 2020-05-13 at 4 05 00 PM" src="https://user-images.githubusercontent.com/15261138/81803435-4fa15580-9535-11ea-985f-42f3cc904704.png">

Any idea?

**Comment by radex** [comment]
i don't think you correctly set up native code as per the instructions - seems like the native module is unavailable

**Comment by piolAtif** [comment]
@radex  Thank you for your reply.  
Previously I used `autolink` to link watemelondb with Xcode. When I Changed it with the manual process. It worked fine.

Xcode - 11.2.1
<img width="859" alt="Screenshot 2020-05-29 at 3 27 21 PM" src="https://user-images.githubusercontent.com/15261138/83247519-42d16280-a1c1-11ea-8f1d-0ad6a5c7b2d7.png">

I faced another issue after this: https://github.com/Nozbe/WatermelonDB/issues/674
solution: `npm uninstall react-native-flipper`. Remove react-native-flipper from your project. 

I hope, it can help someone.
